src/features/todos.ts

```diff
@@ -1,8 +1,16 @@
-import { createSlice } from '@reduxjs/toolkit';
-import { Todo } from '../types/Todo';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Todo } from '../types';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    setTodos: (todos: Todo[], { payload }: PayloadAction<Todo[]>) => [
+      ...todos,
+      ...payload,
+    ],
```

Comment:
The `setTodos` reducer currently concatenates the existing `todos` state with the `payload`. If the intention is to replace the existing todos with the new payload, consider returning only the `payload` instead of concatenating it with the existing state.